apiserver/facades/client/client/perm_test.go

```diff
@@ -265,7 +265,7 @@ func opClientServiceUnexpose(c *gc.C, st api.Connection, mst *state.State) (func
 }
 
 func opClientResolved(c *gc.C, st api.Connection, _ *state.State) (func(), error) {
-	err := st.Client().Resolved("wordpress/1", false)
+	err := st.Client().Resolved("wordpress/1", false, false)
```

Comment:
We are copying the resolved functionality to the Application facade precisely so we don't have to mess with the client facade. There's no need for changes here.